lib/forest_liana/bootstraper.rb

```diff
@@ -100,6 +101,17 @@ def check_integrations_setup
       end
     end
 
+    def namespace_duplicated_models
+      _, models = ForestLiana.models
+        .group_by {|model| model.table_name}
+        .detect { |k, v| v.length > 1 }
+
+      models.each do |model|
```

Comment:
`models` is `nil` here if there is no duplicate model, as a consequence the server crashes on start.